composer.json

```diff
@@ -24,7 +24,7 @@
         "php": "^8.1",
         "ext-json": "*",
         "composer/xdebug-handler": "^3.0",
-        "jetbrains/phpstorm-stubs": "^2022.2|^2023.1",
+        "jetbrains/phpstorm-stubs": "2023.3",
```

Comment:
Sorry, did not notice that my question was pending the whole last week.

Reply:
We use it for the list of classes that come with PHP by default. Like `\Override`.

Since we are now providing scoped release, nothing is requiring us to provide a range to avoid conflict with other dependencies required by the user.

I prefer not to use `@stable` as it can shift under our feet without us noticing, Rather pin to a known working version.

Reply:
Have you had a look inside the scoped stubs?‘vendor/jetbrains/phpstorm-stubs/session/session.php’ (Qossmic/deptrac v2)

Does it still work as expected with the added namespace?

Reply:
Looking at https://github.com/qossmic/deptrac/blob/2.0.x/vendor/jetbrains/phpstorm-stubs/PhpStormStubsMap.php, yes, it will be working just fine with the added namespace.

Reply:
We are looking only at the map file, not at the stubs definitions themselves.

Reply:
Thank you for the clarification!